.editorconfig

```diff
@@ -3,11 +3,9 @@ root = true
 
 [*]
 indent_style = space
-indent_size = 2
```

Comment:
Can you revert these .editorconfig changes and all related whitespace changes?

Reply:
it's possible. What is the reason for reverting white space?